src/xunit.performance.api/XunitPerformanceHarnessOptions.cs

```diff
@@ -104,6 +105,10 @@ public static XunitPerformanceHarnessOptions Parse(string[] args)
                                     Console.WriteLine(Usage());
                                     Environment.Exit(0);
                                     break;
+                                case ErrorType.VersionRequestedError:
```

Comment:
This is executed when the user passes --version to the Harness constructor (request the version of the harness). e.g. `dotnet.exe --version` outputs 1.0.0